compose.yaml

```diff
@@ -6,14 +6,55 @@ services:
 
     build: frontend/fingertips-frontend
     environment:
-      - FINGERTIPS_API_URL=http://api:8080
+      FINGERTIPS_API_URL: "http://api:8080"
     ports:
       - 3000:3000
 
   api:
     profiles:
       - all
       - api
+
+    depends_on:
+      db-setup:
+        condition: service_completed_successfully
```

Comment:
This condition means that the API doesn't start until the `db-setup` container has run successfully. It means that the API doesn't start until the DB is ready.